simpleclient_jetty/src/main/java/io/prometheus/client/jetty/JettyStatisticsCollector.java

```diff
@@ -0,0 +1,81 @@
+package io.prometheus.client.jetty;
+
+import io.prometheus.client.Collector;
+import org.eclipse.jetty.server.handler.StatisticsHandler;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Collect metrics from jettty's org.eclipse.jetty.server.handler.StatisticsHandler.
+ */
+public class JettyStatisticsCollector extends Collector {
+  private final StatisticsHandler statisticsHandler;
+  private static final List<String> EMPTY_LIST = new ArrayList<String>();
+
+  public JettyStatisticsCollector(StatisticsHandler statisticsHandler) {
+    this.statisticsHandler = statisticsHandler;
+  }
+
+  @Override
+  public List<MetricFamilySamples> collect() {
+    return Arrays.asList(
+            buildGauge("jetty_requests", "number of requests", statisticsHandler.getRequests()),
```

Comment:
This should be a Counter then with a _total suffix, or combine with request_time to produce a summary.